app/models/user.rb

```diff
@@ -43,6 +44,6 @@ def leader_of
 
   validates :email, presence: true
 
-  has_attached_file :photo, :styles => { :medium => "500x500>", :thumb => "150x150#" }, :default_url => "/images/:style/missing.png"
+  has_attached_file :photo, :styles => { :medium => "500x500>", :thumb => "150x150#" }, default_url: "thumb/default.png"
```

Comment:
@andylouisqin you're right, for some reason I was under the impression that we needed to have both styles of placeholder default photos, but it ImageMagick auto generates these then I'll delete the smaller image :)